user-docs/documentation/how-to/6-configure-ci-cd.md

```diff
@@ -3,3 +3,198 @@ sidebar_position: 6
 ---
 
 # Configure your CI/CD
+
+## Choose the correct IAM policy to give your CI user
```

Comment:
If using github actions, we can recommend to follow these [guidelines to setup OIDC](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services), removing the need to store long lived credentials, and this removing the need to specify a policy (we can give admin access)